.github/actions/patch-lifecycle-manager/action.yaml

```diff
@@ -11,7 +11,7 @@ runs:
         echo \
         "- op: add
           path: /spec/template/spec/containers/0/args/-
-          value: --watcher-dev-registry=true
+          value: --skr-watcher-image-registry=europe-docker.pkg.dev/kyma-project/dev
```

Comment:
This suffers from the same problem as described in this PR: https://github.com/kyma-project/lifecycle-manager/pull/1868

The E2E tests fail because it tries to apply the old flag instead of the new one. If KLM comes up, it will use the default which is the PROD registry. This one does not contain the PR build images.